### PR TITLE
Updated Upstream (CraftBukkit)

### PR DESCRIPTION
**Pull request by Machine-Maker**
Upstream has released updates that appear to apply and compile correctly.
This update has not been tested by PaperMC and as with ANY update, please do your own testing

CraftBukkit Changes:
51e2981b #831: Reload unloaded main worlds correctly

### CODE REVIEW

File: Spigot-Server-Patches/0690-Implement-Keyed-on-World.patch [hunk 1]
```diff
@@ -64,19 +64,19 @@ index 760579921927b4c8b0f20b2611b95fd626e4b27f..3075700dfa992da81b10246fcf7c7ad1
...
 --- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
 +++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
-@@ -1154,7 +1154,7 @@ public final class CraftServer implements Server {
-             chunkgenerator = worlddimension.c();
+@@ -1161,7 +1161,7 @@ public final class CraftServer implements Server {
+         } else if (name.equals(levelName + "_the_end")) {
+             worldKey = net.minecraft.world.level.World.THE_END;
+         } else {
+-            worldKey = ResourceKey.a(IRegistry.L, new MinecraftKey(name.toLowerCase(java.util.Locale.ENGLISH)));
++            worldKey = ResourceKey.newResourceKey(IRegistry.getWorldRegistry(), new net.minecraft.resources.MinecraftKey(creator.key().getNamespace().toLowerCase(java.util.Locale.ROOT), creator.key().getKey().toLowerCase(java.util.Locale.ROOT))); // Paper
```
Comment (Proximyst) [inline Spigot-Server-Patches/0690-Implement-Keyed-on-World.patch]: English locale

**Review by Proximyst — DISMISSED** [review]

File: Spigot-Server-Patches/0690-Implement-Keyed-on-World.patch (hunk 1 above)
Comment (Machine-Maker) [reply Spigot-Server-Patches/0690-Implement-Keyed-on-World.patch]: oh yeah, I always forget to change it from what intellij autocompletes to.

Comment (Machine-Maker) [reply Spigot-Server-Patches/0690-Implement-Keyed-on-World.patch]: Done

**Review by MiniDigger — APPROVED** [review]

**Comment by kennytv** [comment]
:AlienPls3:

**Pull request merged**